src/ui/templates/cards/accordion.hbs

```diff
@@ -11,7 +11,7 @@ id="{{id}}">
         {{{_config.title}}}
       </h3>
       <div class="yxt-AccordionCard-icon js-yxt-AccordionCard-icon{{#if isExpanded}} yxt-AccordionCard-icon--expanded{{/if}}">
-        {{> icons/iconPartial iconName='chevron'}}
+        {{> icons/bakedInIcon iconName='chevron'}}
```

Comment:
nit: What do you think of 'builtInIcon' instead?

Reply:
I used bakedInIcon because builtInIcon was hard for my eyes to read, specifically the bu`iltInIc`on part

Reply:
ya, I actually noticed that when I typed it out. As-is looks good to me

Reply:
bakedInIcon is also not perfect because of the d and I looking similar, but it also seemed like a slightly more flavorful word